### ISSUE TITLE
Changing the accent color affects some themes

### ISSUE THREAD

**Issue by Kokecena** [JFormDesigner/FlatLaf#625]
Hello, I recently added the function to change the accent color to my program, but I just noticed that they apply to some themes too, I thought this only applied to core themes, should it work like this? Or is it a special function that I missed haha 😄, I see that the same thing happens in the demo application, Is there a way to avoid it and only apply it to core themes?

Before to change accent color
![before](https://user-images.githubusercontent.com/75457600/206322151-9e4b393b-f7c4-4707-a9b4-ee1a3d1fbd11.PNG)

After
![after](https://user-images.githubusercontent.com/75457600/206322180-c834d72e-00c2-4612-ab9f-31e6d0b5276f.PNG)

**Comment by DevCharly** [comment]
Thanks for reporting.

Fixed in latest `3.1-SNAPSHOT`: https://github.com/JFormDesigner/FlatLaf#snapshots

Note: the fix works only if using `FlatLaf.setSystemColorGetter()` (available since FlatLaf 3)  to change accent color. E.g.:

~~~java
FlatLaf.setSystemColorGetter( name -> {
    return name.equals( "accent" ) ? myAccentColor : null;
} );
~~~

**Comment by Kokecena** [comment]
I just tried the snapshot and using `FlatLaf.setSystemColorGetter()` and it doesn't happen anymore.


![test1](https://user-images.githubusercontent.com/75457600/213840582-dee8dafc-48c8-411b-9e00-af356a6b0e70.PNG)

![test2](https://user-images.githubusercontent.com/75457600/213840588-c903c8c7-3665-4e28-9745-5970bdc57a20.PNG)

Thank you for such an incredible project.
